kfs-ld/src/main/java/edu/arizona/kfs/module/ld/LaborConstants.java

```diff
@@ -25,4 +25,7 @@
 	
 	public static final String[] BALANCE_TYPES = {"AC", "CB", "IE"};
 
+	//SalaryExpenseTransferDocument
+	public static final String BALANCE_TYPE_ACTUAL = "AC";
+	public static final String BALANCE_TYPE_A2 = "A2";
```

Comment:
This change was implemented. I noticed that code A2 does not exist on the System Options table (FS_OPTION_T), so I kept the constant for Balance Type A2.